ns-provisioner-samples/gitops-airgap/install/namespaces.yaml

```diff
@@ -0,0 +1,15 @@
+#@ load("@ytt:data", "data")
+#! This for loop will loop over the namespace list in desired-namespaces.yaml and will create those namespaces.
+#! NOTE: if you have another tool like Tanzu Mission Control or some other process that is taking care of creating namespaces for you, 
+#! and you don’t want namespace provisioner to create the namespaces, you can delete this file from your GitOps install repository.
+#@ for ns in data.values.namespaces:
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: #@ ns.name
+  annotations:
+    kapp.k14s.io/create-strategy: fallback-on-update
```

Comment:
What are these kapp annotations? do we need this?

Reply:
If the namespaces is manually created we own it